spec/filters_spec.rb

```diff
@@ -155,7 +155,9 @@ def matches?(_)
       end
     end
 
-    context "with the default profile" do
+    context "with the default configuration" do
+      skip "requires the default configuration" if ENV["SIMPLECOV_NO_DEFAULTS"]
```

Comment:
Slightly confused why this is here and not just above? Leftover? :)

Reply:
I'm not sure if there's a scenario where you'd run the specs w/ SIMPLECOV_NO_DEFAULTS, but this should ensure the suite still passes in that case, since it has to do with the default filters. Might be worth setting up in Travis?